tests/e2e/migration.spec.js

```diff
@@ -0,0 +1,99 @@
+const _ = require("lodash");
+const ExtensionUtil = require("./util");
+const { MiscURLs, StoreKey } = require("../../src/js/shared/constants");
+const ConfigFile = require("../../src/js/shared/config_file").default;
+const DefaultFile = require("../../settings.json");
+
+describe("Migration", () => {
+  let browser;
+  let page;
+
+  beforeAll(async () => {
+    browser = await ExtensionUtil.load();
+  });
+
+  test("Data from local storage is copied to chrome's storage", async () => {
+    // Go to options page to set sample data to local storage
+    page = await ExtensionUtil.goto("options");
+
+    const sampleData = {
+      key: "_configUrl",
+      value: "test value",
+    };
```

Comment:
I should have been more clear on the issue, but here you are just checking that `_configUrl` is being migrated, and you should check that **all** values (used on v4) are migrated (e.g. `_configLastRefresh`, `_configUseGroups`, `_configEnc`, `_configAutoRefresh`, `_group1Name`, `_allsearch`, `_RSAConfig`, `_RSAallquery`, and so on)